assets.did

```diff
@@ -59,6 +59,7 @@ type HttpResponse = record {
 type StreamingCallbackHttpResponse = record {
   body: blob;
   token: opt StreamingCallbackToken;
+  chunk_tree: opt text;
 };
```

Comment:
I'm surprised to see `text` here. What is this? Is there a high-level specification for this feature?

Reply:
This is the serialized tree witness and the result of executing the [function](https://github.com/dfinity/cdk-rs/pull/219/files#diff-6c47c7e1d7716f550a6e35efa3c5e45b6e9737a0ac2fbbb879f04f2753d84a45R1092)

Reply:
But why is it a `text`? You say serialized, so why not a `blob`? Also, Candid is already a serialization format, so one might want to use Candid to encode the witnesss?

But code review isn’t the right place to discuss the design and specificatoin of the feature. We really need to have a spec for the HTTP gateway feature where we can design extensions like this.

Is anyone at DFINITY planning to do that? Maybe a separate file in the interface-spec repo, or even part of the interface spec? (@Dfinity-Bjoern?)

Reply:
changed to `opt blob`